### ISSUE TITLE
Zalando-specific?

### ISSUE THREAD

**Issue by LappleApple** [zalando/rant-gate#1]
Hey, this seems pretty Z-specific. Is it a candidate for true open-source?

**Comment by JonasJurczok** [comment]
Well. The initial phase is pretty zalando specific but in the end it really depends on the configuration. The same way as stups is a framework for running clouds this should turn into a framework for running onboarding (insert big plan buzword phrase here :) ).

So if everything works out this will be one of a couple of RANT services that together form a system to help you do the onboarding process.

If you want to discuss this feel free to approach me :)

**Comment by LappleApple** [comment]
Thanks for the info, Jonas. In the spirit of Open Source First I'm wondering if this project might be best left internal for now, and go public when the community can use it.

**Comment by JonasJurczok** [comment]
Yes, I thought about that to use our internal CI/CD solutions anyway. As soon as we are continuing active development on this I will move it back to internal repos.
